analytics-core/src/test/java/com/segment/analytics/TestUtils.java

```diff
@@ -12,6 +15,10 @@ private TestUtils() {
     throw new AssertionError("No instances.");
   }
 
+  public static final Gson GSON = new GsonBuilder()
```

Comment:
Since this is only used in the `ISO8601DateAdapterTest`, I think we can move this into that file for now.

Reply:
Makes sense, I'll move it over.